tests/unit/slices/filters.ts

```diff
@@ -101,6 +101,44 @@ describe('filter slice reducer works as expected', () => {
     expect(actualState).toEqual(selectedExpectedState);
   });
 
+  it('setFilterOption action is handled properly with a Filter with a NumberRangeValue', () => {
+    const selectedFilter: SelectableFilter = {
```

Comment:
can some of these filter fields be defined at the start of the test suite so they can be used by both number range tests? it seems like they're mostly the same in both